### DISCUSSION TITLE
Plu Discussion with 짠의민

### DISCUSSION THREAD

**Discussion by kimscastle** [Team-Plu/Plu-iOS#57] [Announcements]
## Discusstion Rule
- Discussion에 대한 rule을 개인적으로 정해봤습니다

### Announcements
- 이건 전체적으로 일정이나 공지를 띄울때 사용합니다

### Idea
- 합의되지 않은 것들중에서 이야기를 들어보고 싶은 주제가있으면 해당 category를 채택합니다

### General
- 합의된것들중에서 기록으로 남길만한것들이있으면 해당 category를 채택합니다
- 예를들어서 `alamofire의 네트워크레이어 레퍼런스를 모아봅시다`는 합의된 내용이고 결정하는 과정을 기록하고싶은 내용이에 general을 채택합니다

### Q&A
- 카톡방에 물어보기전에 해당 카테고리로 discussion을 올리고 카톡방에 `q&a확인해주세요`정도로 통일하면 좋을거같아요
- 해당 질문에대한 기록을 남겨놓는게 좋을거같아요

### Labels
- Labels 경우엔 민재,찬미,의성만 선택합니다

**Comment by kimscastle** [comment]
제가 개인적으로 생각한 룰이 이정도라

- 추가로 이런 카테고리가있으면 좋겠다
- 이런거 기록했으면 좋겠다

하는게 있으면 자유롭게 코멘트 달아주세요
@cchanmi @ffalswo2

**Reply by ffalswo2** [reply]
좋습니다:) 추후에 또 떠오르는 아이디어가 있으면 코멘트 남기겠습니다.

**Reply by cchanmi** [reply]
좋아요 확인했숩니다
